packages/woff2/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/google/woff2/tree/v1.0.2
```

Comment:
No better homepage than this? If so, put the version definition first and then use it here to define the homepage, so it stays up to date.

Reply:
https://www.w3.org/TR/WOFF2/ better ?